src/ViewArticle/ViewArticleStyles/ViewArticle.scss

```diff
@@ -0,0 +1,95 @@
+@import '../../commons/assets/variables/font.scss';
+@import '../../commons/assets/variables/colors.scss';
+
+.article_title{
+    color: #00a4d3;
+    font-family:$source-san-font;
+    font-weight: light;
+    font-size: 40px;
+    margin-top:5px;
+}
+.myimage{
+    width: 90%;
+    margin-left: 5%;
```

Comment:
@jennizalwango instead of using `margin-left:50% `and `margin-right:5%`. I suggest you instead just set `margin: auto` as this will balance the element well everywhere in your style file.

Reply:
okay let me work on it